modules/assimp/editor_scene_importer_assimp.cpp

```diff
@@ -28,24 +28,25 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
-#include "assimp/DefaultLogger.hpp"
-#include "assimp/Importer.hpp"
-#include "assimp/LogStream.hpp"
-#include "assimp/Logger.hpp"
-#include "assimp/SceneCombiner.h"
-#include "assimp/cexport.h"
-#include "assimp/cimport.h"
-#include "assimp/matrix4x4.h"
-#include "assimp/pbrmaterial.h"
-#include "assimp/postprocess.h"
-#include "assimp/scene.h"
+#include <assimp/SceneCombiner.h>
```

Comment:
Not a big deal, but as per our style guide these should come after Godot's own headers.

`#include "editor_scene_importer_assimp.h"` should also be the first incldue.

http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes

Reply:
"zutil.h" below is also a thirdparty header btw, should have the same treatment.